src/style.css

```diff
@@ -1,3 +1,81 @@
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-link:#00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+nav {
+  display: block;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+    height: 40px;
+    width: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
```

Comment:
Just a quick thing I forgot, avoid using horizontal margin, use padding!

Reply:
Hello @jv-aquino , i made the changes.